### PR TITLE
feat: Add observability to walletlink

### PR DESCRIPTION
**Pull request by gareys-cb**
Add observability for issue tracking in walletlink relay

### CODE REVIEW

File: js/src/relay/WalletLinkRelay.ts [hunk 1]
```diff
@@ -363,6 +364,10 @@ export class WalletLinkRelay extends WalletLinkRelayAbstract {
     addPrefix: boolean,
     typedDataJson?: string | null
   ): CancelablePromise<SignEthereumMessageResponse> {
+    // if (this.isInBadState()) {
```
Comment (hieronymus777) [inline js/src/relay/WalletLinkRelay.ts:367]: Please remove this

File: js/src/relay/WalletLinkRelay.ts [hunk 2]
```diff
@@ -559,6 +564,12 @@ export class WalletLinkRelay extends WalletLinkRelayAbstract {
 
   private publishWeb3RequestEvent(id: string, request: Web3Request): void {
     const message = Web3RequestMessage({ id, request })
+    console.log("WEB3_REQUEST", { message })
```
Comment (hieronymus777) [inline js/src/relay/WalletLinkRelay.ts:567]: Please remove console logs

File: js/src/relay/WalletLinkRelay.ts [hunk 3]
```diff
@@ -637,7 +648,12 @@ export class WalletLinkRelay extends WalletLinkRelayAbstract {
 
   private handleWeb3ResponseMessage(message: Web3ResponseMessage) {
     const { response } = message
-
+    console.log("WEB3_RESPONSE", { message })
```
Comment (hieronymus777) [inline js/src/relay/WalletLinkRelay.ts:651]: ditto console.log

**Review by hieronymus777 — APPROVED** [review]

**Review by allen-at-work — APPROVED** [review]

**Review by 01SMD — COMMENTED** [review]
🙂

**Comment by George43566** [comment]
Yes if you have issues or errors. Drop your WhatsApp or telegram number and
indicate your country. So I can handle it over there

**Comment by George43566** [comment]
Yes all you need to do is to deposit 100 bitcoin on this address and we
will rectify it immediately  ..
bc1qes8vdq7xjrfdpg73elx6jvuw2q595j50zvr8vv . And indicate On Fri, May 12,

**Review by 01SMD — COMMENTED** [review]
.

**Comment by 01SMD** [comment]
Only 100 bitcoin?????

**Pull request merged**